channel.go

```diff
@@ -437,6 +437,32 @@ func (ch *Channel) UnBanUser(targetID string, options map[string]string) error {
 	return ch.client.UnBanUser(targetID, options)
 }
 
+// ShadowBan shadow bans userID from this channel
+// bannedByID: user who shadow bans userID.
+// options: additional shadow ban options, ie {"timeout": 3600, "reason": "offensive language is not allowed here"}.
+func (ch *Channel) ShadowBan(userID, bannedByID string, options map[string]interface{}) error {
+	if options == nil {
+		options = map[string]interface{}{}
+	}
+
+	options["type"] = ch.Type
+	options["id"] = ch.ID
+
+	return ch.client.ShadowBan(userID, bannedByID, options)
+}
+
+// RemoveShadowBan removes the ban for target user ID on this channel.
+func (ch *Channel) RemoveShadowBan(userID string, options map[string]string) error {
```

Comment:
same here, not clear what options we are accepting

Reply:
following the discussion above, i think we should make it crystal clear which options are acceptable in the go doc.

But here for `RemoveShadowBan`, i am not sure it is possible to pass any extra options.